content/frontmatter.tex

```diff
@@ -15,7 +15,7 @@
 \input{content/coverpage}
 
 \setcounter{tocdepth}{4}
-\setcounter{secnumdepth}{3}
+\setcounter{secnumdepth}{4}
```

Comment:
@BryantLam Can we go back to depth 3 ? It doesn't render well. Also, I think reductions with various operations doesn't have to be called out in the TOC.

Reply:
@BryantLam I will leave it for now. I will post the pdf, since we have only a few hours for the deadline. We can handle this as DocEdit, if need be.

Reply:
@manjugv The intent for why I changed the [`\textbf` to `\paragraph`](https://github.com/openshmem-org/specification/pull/113/files#diff-2e96c86806771c136a465183e2647ce8R9) was to specifically render a TOC entry for each reduction operation. I thought it would be a nice convenience to create some jump points to specific reduction operations. If there's consensus against this, we can remove it (but I'll be sad).